ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -16,7 +16,7 @@
 
     private static final String URL = "/admin/index.htm";
 
-    private static final By MANAGE_USERS = By.cssSelector("#legacyui-manageUsers a");
+    private static final By MANAGE_USERS = By.cssSelector("#content a[href='/openmrs/admin/users/users.list']");
```

Comment:
Inspection Lab report indicates that the current property attribute for the [element](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/285dcc90731099bd2d9368097428a21b3c8bcca2/ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java#L19) does not exist. Apart from the suggested property, the element can also be retrieved via its `id, xpath, linkText and others` but in terms of` elementLoadLoadTime`, the suggested one takes less time to retrieve the element than any other attribute can do thus overriding the rest.